pkg/controller/vsphere/reconciler_test.go

```diff
@@ -2610,13 +2626,23 @@ func TestCreate(t *testing.T) {
 
 			client := builder.Build()
 
+			gates, err := testutils.NewDefaultMutableFeatureGate()
+			if err != nil {
+				t.Errorf("Unexpected error setting up feature gates: %v", err)
+			}
+
+			err = gates.Set(fmt.Sprintf("%v=%v", features.FeatureGateVSphereStaticIPs, tc.staticIPFeatureGateEnabled))
+			if err != nil {
```

Comment:
Nit, should inline this
```suggestion
			if err := gates.Set(fmt.Sprintf("%v=%v", features.FeatureGateVSphereStaticIPs, tc.staticIPFeatureGateEnabled)); err != nil {
```

Reply:
Ok, I'll update that.